### ISSUE TITLE
Problems running "Simple introduction tutorial"

### ISSUE THREAD

**Issue by SergeyYakubov** [Libensemble/libensemble#1167]
trying to run the tutorial from https://libensemble.readthedocs.io/en/main/tutorials/local_sine_tutorial.html,
I get errors like

```   
from libensemble import Ensemble, LibeSpecs, SimSpecs, GenSpecs, ExitCriteria
ImportError: cannot import name 'LibeSpecs' from 'libensemble' (/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/libensemble/__init__.py)

```
fixing error with imports (use `from libensemble.specs import LibeSpecs, SimSpecs, GenSpecs,  ExitCriteria
`)

I got another error:

```
File "/Users/35y/projects/reviews/libEnsemble/calling_script.py", line 31, in <module>
    ensemble.add_random_streams()  # setup the random streams unique to each worker
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/libensemble/ensemble.py", line 535, in add_random_streams
    nstreams = self._nworkers()
  File "/Library/Frameworks/Python.framework/Versions/3.10/lib/python3.10/site-packages/libensemble/ensemble.py", line 396, in _nworkers
    return self.libE_specs.nworkers
AttributeError: 'ExitCriteria' object has no attribute 'nworkers'

```

so, I guess the tutorial has to be updated for the latest libEnsemble version.

**Comment by jlnav** [comment]
Hi @SergeyYakubov ,

Whoops, sorry about that! Thank you for opening this issue to point that out, we're fixing those tutorial bugs and any others that may show up in #1168 .

Please let us know if there's anything else you need!

**Comment by shuds13** [comment]
@SergeyYakubov 

The tutorial has been fixed. Everything should work now.

Thanks for finding this.

**Comment by jlnav** [comment]
Closed by #1171
